absl/base/attributes.h

```diff
@@ -602,7 +602,9 @@
 // duration, because it guarantees that they will not suffer from the so-called
 // "static init order fiasco".  Prefer to put this attribute on the most visible
 // declaration of the variable, if there's more than one, because code that
-// accesses the variable can then use the attribute for optimization.
+// accesses the variable can then use the attribute for optimization. In the
+// case of extern variables, the annotation should also be placed on the
```

Comment:
How about "If that declaration is not visible at the definition (i.e. an `extern` variable), then the definition must also be marked `ABSL_CONST_INIT` so the compiler knows to enforce constant initialization."

I think what you're trying to add to the documentation is correct and important, but I want to avoid confusing Googlers, who need to use ABSL_CONST_INIT fairly frequently (due to style guide requirements) but will almost never encounter this particular situation (due to style guide constraints).

Reply:
Thanks, that sounds much clearer.  Updated the PR.